waku/waku.md

```diff
@@ -256,15 +258,27 @@ The projection function is defined as a mapping from a 4-byte slice S to a 512-b
 	D[n] = 1
 	END FOR
 
-#### P2P Request
+A full bloom filter means that the node is to be considered a `Full Node` and it will accept any topic.
+If both Topic Interest and Bloom filter are specified, Topic Interest always takes precedence and Bloom Filter MUST be ignored.
+If only Bloom Filter is specified, the current Topic Interest MUST be discarded and only the updated Bloom Filter MUST be used when forwarding envelopes.
 
-This packet is used for sending Dapp-level peer-to-peer requests, e.g. Waku Mail Client requesting old messages from the Waku Mail Server.
+An bloom filter of all 0s signals that the node is not currently interested in receiving any envelope.
```

Comment:
```suggestion
A bloom filter with all bits set to 0 signals that the node is not currently interested in receiving any envelope.
```

Reply:
Might make sense sense to state that all set to 1s indicate full node here